parachains/runtimes/assets/statemint/src/weights/pallet_assets.rs

```diff
@@ -67,13 +69,12 @@ impl<T: frame_system::Config> pallet_assets::WeightInfo for WeightInfo<T> {
 	/// The range of component `s` is `[0, 5000]`.
 	/// The range of component `a` is `[0, 500]`.
 	fn destroy(c: u32, s: u32, a: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 96_000
-			.saturating_add(Weight::from_ref_time(15_834_000 as u64).saturating_mul(c as u64))
-			// Standard Error: 96_000
-			.saturating_add(Weight::from_ref_time(17_275_000 as u64).saturating_mul(s as u64))
-			// Standard Error: 966_000
-			.saturating_add(Weight::from_ref_time(35_635_000 as u64).saturating_mul(a as u64))
+		// Minimum execution time: 74_325_463 nanoseconds.
+		Weight::from_ref_time(74_947_012_000 as u64)
```

Comment:
One more heavy jump

Reply:
Good to see we all spotted it. It's just that one call but on all three runtimes.